src/planscape/planning/serializers.py

```diff
@@ -88,6 +90,36 @@ class Meta:
         geo_field = "geometry"
 
 
+class ValidatePlanningAreaSerializer(gis_serializers.GeoModelSerializer):
+    geometry = gis_serializers.GeometryField()
+
+    def validate_geometry(self, geometry):
+        if not isinstance(geometry, GEOSGeometry):
+            geometry = GEOSGeometry(
+                geometry,
+                srid=settings.CRS_INTERNAL_REPRESENTATION,
+            )
+
+        if geometry.srid != settings.CRS_INTERNAL_REPRESENTATION:
+            geometry = geometry.transform(
+                settings.CRS_INTERNAL_REPRESENTATION, clone=True
+            )
+
+        try:
+            geometry = coerce_geometry(geometry)
+        except ValueError as valEx:
+            raise serializers.ValidationError(str(valEx))
+        return geometry
+
+    class Meta:
+        model = PlanningArea
+        fields = ("geometry",)
+
+
+class PlanningAreaValidationSerializer(serializers.Serializer):
```

Comment:
This name seems easy to confuse with `ValidatePlanningAreaSerializer` -- if the main purpose of this is to be an out serializer that includes area_acres, could we name it something that clarifies this?